projects/xpdf/fuzz_pdfload.cc

```diff
@@ -116,6 +117,13 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
             (void)splashOut->getBitmap();
 
             delete splashOut;
+
+            Object globals;
+            BaseStream *base_str = doc.getBaseStream();
```

Comment:
Can this ever be `nullptr`, and can `JBIG2Stream` handle that if so?

Reply:
here adds the check: https://github.com/google/oss-fuzz/pull/7476